HPXMLtoOpenStudio/resources/output.rb

```diff
@@ -1160,6 +1306,85 @@ def self.append_sizing_results(hpxml_bldgs, results_out)
     return results_out
   end
 
+  # Appends electric panel results to the provided array for use in writing output files.
+  #
+  # @param hpxml_bldgs [Array<HPXML::Building>] List of HPXML Building objects representing an individual dwelling unit
+  # @param results_out [Array] Rows of output data
+  # @return [Array] Rows of output data, with electric panel results appended
+  def self.append_panel_results(hpxml_header, hpxml_bldgs)
```

Comment:
Looks like this method needs a few minor updates:

```suggestion
  # Returns electric panel results for use in writing output files.
  #
  # @param hpxml_bldgs [Array<HPXML::Building>] List of HPXML Building objects representing an individual dwelling unit
  # @return [Array] Rows of output data, with electric panel results appended
  def self.get_panel_results(hpxml_header, hpxml_bldgs)
```